app/components/Footer.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from "react";
+
+const Footer: FC = () => {
+  const termsOfService: string = "https://stellar.org/terms-of-service";
+  const privacyPolicy: string = "https://stellar.org/privacy-policy";
+  const gitHubLink: string = "https://github.com/MatHenriquez/stellar-wallet";
```

Comment:
This should come from an environment variable.

Reply:
I have moved these links to the environment variables file and imported them from there